components/Sidebar/sidebar_data.ts

```diff
@@ -0,0 +1,71 @@
+export const sidebar_data: SidebarData[] = [
```

Comment:
move this file to `constants/sidebarData.tsx`. stick to camel case for naming files